lib/fpmon.gi

```diff
@@ -210,7 +210,7 @@ function( F, rels )
     if Length(gens) > Length(rels) then
       SetIsFinite(s, false);
     fi;
-	
+        
```

Comment:
I changed all the tabs in this file to spaces in a single commit, then fixed the function in a separate commit. So, no there are no actual changes up to this point.